AEPMedia/Sources/Media.swift

```diff
@@ -119,6 +120,14 @@ public class Media: NSObject, Extension {
             return
         }
 
-        tracker.track(eventData: event.data)
+        tracker.track(event: event)
+    }
+
+    private func dispathSessionCreated(eventData: [String: Any]) {
```

Comment:
Question: Do we always get sessionCreate with or without Assurance?

Reply:
Yes. We will dispatch this event even without active assurance session.